Projekt/Projekt/ClientMessage/ClientMessage/Program.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Text;
+using System.Net;
+using System.Net.Sockets;
+
+namespace ClientMessage
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //Vælg om du skal være klient eller server
+            Console.WriteLine("Skriv 'Server' eller 'Klient'");
+            string ClientOrServer = Console.ReadLine().ToLower().Trim();
+
+            if (ClientOrServer.Contains("server"))
```

Comment:
Dette er en meget lang if sætning. For bedre læsevenlighed bør i dele programmet op i funktioner. Evt. en Klient funktion og en Server funktion